src/node.cc

```diff
@@ -4795,7 +4799,25 @@ Local<Context> NewContext(Isolate* isolate,
   return context;
 }
 
+inline static bool TickEventLoop(Environment & env) {
+  bool more;
+  uv_run(env.event_loop(), UV_RUN_ONCE);
```

Comment:
TODO: I changed the second parameter to `UV_RUN_ONCE`, which was `UV_RUN_DEFAULT` before.

Reply:
Maybe UV_RUN_NOWAIT is a better alternative here:
> UV_RUN_NOWAIT: Poll for i/o once but don’t block if there are no pending callbacks. Returns zero if done (no active handles or requests left), or non-zero if more callbacks are expected (meaning you should run the event loop again sometime in the future). (from the [doc uv](http://docs.libuv.org/en/v1.x/loop.html#c.uv_run))

With UV_RUN_ONCE, UV waits until some event happens and handles it before returning, meaning it could block our code indefinitely (at least that's how I understood it...)